### DISCUSSION TITLE
示例问题：做到什么程度才算优秀的程序员?

### DISCUSSION THREAD

**Discussion by zxuqian** [zxuqian/frontend-questions#1] [求助]
来自 unknown_ZH@bilibili

**Comment by zxuqian** [comment]
优秀是相对的。你自己认为优秀就优秀了，不用考虑太多别人的感受。一般我认为能做到给你一个需求，能自己独立实现，然后能在后边慢慢的优化、重构代码，善于从错误中提高自己的都是优秀的程序员。（要记住，没有绝对优秀的程序员，只要你肯比别人下功夫，就是优秀的。）

**Comment by wayne214** [comment]
需要持续学习提升自己的能力，优秀没有止境，遇到问题，能够找到方法解决，我觉得就是优秀了
